content/pandas/concepts/built-in-functions/terms/get-dummies/get-dummies.md

```diff
@@ -0,0 +1,54 @@
+Subjects: # Please only use Subjects in the subjects.md file (https://github.com/Codecademy/docs/blob/main/documentation/subjects.md). If that list feels insufficient, feel free to create a new Subject and add it to subjects.md in your PR!
+  - 'Data Science'
+  - 'Machine Learning'
+Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
+  - 'Data'
+  - 'Encoding'
+  - 'Pandas'
+  - 'Scikit-learn'
+  - 'Statistics'
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
+  - 'path/data-scientist'
+---
+
+The **`.get_dummies()`** function creates dummy (indicator) variables from categorical variables. Each level of the categorical variable gets its own column - a dummy variable. The dummy variables take on the value "1" if the original categorical variable was that level or "0" if not. 
+
+## Syntax
+
+```pseudo
+pd.get_dummies(data)
+```
+
+The `data` parameter is the data to be converted to dummy variables. It is the only mandatory parameter. It can be any array-like data structure including a pandas Series or DataFrame. 
+
+A number of other parameters are optional or have default arguments. Some of the key parameters are listed below.
+
+| Parameter Name | Data Type | Usage |
+| :--- | :--- | :--- |
+| `prefix` | str, list of str, or dict of str, default None | String to append to the beginning of DataFrame column names. If a list is passed, its length should be equal to the number of columns. A dictionary can also be passed; it should map column names to prefixes. |
+| `prefix_sep` | str, default '_' | If the `prefix` parameter is not `None`, this is a string appended after the prefix separating it from the level of the categorical variable. |
+| `dummy_na` | bool, default False | Adds a column to indicate NaNs. If false, NaNs are ignored. |
+| `columns` | list-like, default None | If a DataFrame is passed to the `data` parameter, a list of columns to can be passed to the `columns` parameter to be encoded as dummy variables. If `columns` is `None` then all the columns with object, string, or category dtype will be converted. |
+| `drop_first` | bool, default False | If `True`, this drops the first level of each variable to undergo dummy encoding. | 
+
+## Codebyte Example
+
+Creates a list of letters in the alphabet `abcs`, converts `abcs` to a Series, and prints the Series. Then uses the `.get_dummys()` function on the Series to create a DataFrame whose columns are indicator variables for each level of the single categorical variable in the Series. Finally, prints the DataFrame. 
```

Comment:
```suggestion
The code below creates a list based on the letters of the alphabet `abcs`, converts `abcs` to a Series, and prints the Series. Then the `.get_dummys()` function is applied to the Series to create a DataFrame whose columns are indicator variables for each level of the single categorical variable in the Series. Finally, the DataFrame is printed. 
```

Reply:
For future entries please try to develop original content for the examples. I think this is different enough from the pandas documentation to be included as is, but ideally we would prefer something unlike what's available on other sites.